pkgs/servers/plexpass/default.nix

```diff
@@ -0,0 +1,12 @@
+{ stdenv, fetchurl, plex }:
+
+stdenv.lib.overrideDerivation plex (attrs: rec {
+  name = "plexpass-${version}";
+  version = "0.9.15.3.1674";
+  vsnHash = "f46e7e6";
+
+  src = fetchurl {
+    url = "https://downloads.plex.tv/plex-media-server/${version}-${vsnHash}/plexmediaserver-${version}-${vsnHash}.x86_64.rpm";
+    sha256 = "086njnjcmknmbn90mmvf60ls7q73g2m955yk621jjdngs4ybvm19";
+  };
```

Comment:
When I copy the meta section from the source `plex` definition and make relevant modifications, I get this error:

```
error: cannot coerce a set to a string, at /nixpkgs/pkgs/servers/plexpass/default.nix:4:3
```

When I resolve it, I'll update this PR. If you have any hints, I'd appreciate it. Thanks.